lib/vmpooler/pool_manager.rb

```diff
@@ -455,6 +455,31 @@ def _check_snapshot_queue
       end
     end
 
+    def migrate_vm(vm, pool)
+      Thread.new do
+        _migrate_vm(vm, pool)
+      end
+    end
+
+    def _migrate_vm(vm, pool)
+      $redis.srem('vmpooler__migrating__' + pool, vm)
+      vm_object = $vsphere[pool].find_vm(vm) || $vsphere[pool].find_vm_heavy(vm)
+      host = $vsphere[pool].find_least_used_compatible_host(vm_object)
+      parent_host = vm_object.summary.runtime.host
+      parent_host_name = parent_host.name
+      if host == parent_host
+        $logger.log('s', '[ ] [' + pool + "] No migration required for '" + vm + "'")
+      else
+        start = Time.now
```

Comment:
This has been added. It looks like it follows the currently used patterns, but please let me know if I've missed something or if it needs further adjustment.